Handler/Docs.hs

```diff
@@ -1,19 +1,13 @@
 module Handler.Docs where
 
 import Import
+import Model.Site
 
 getDocsR :: Handler Html
 getDocsR = do
     root <- getAppRoot
-    Entity siteId _ <- runDB $ upsert (demoSite root) []
+    siteId <- runDB $ upsertDemoSite root
```

Comment:
We could make this into a `Widget` so the function/template pair can find the `siteId` on its own, right?